src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  --main-color: #00acdc;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
```

Comment:
no needs to hardcode height for elements with content inside
Just make your links as `inline-block` with top/bottom `padding` 23px
```suggestion
```